package.json

```diff
@@ -1,35 +1,48 @@
 {
   "name": "fnlint",
-  "version": "0.1.1",
-  "description": "Lint file name formatting",
-  "main": "lib/fnlint.js",
-  "engines": {
-    "node": ">=4.0.0"
-  },
-  "scripts": {
-    "test": "./scripts/test.sh"
-  },
+  "version": "0.0.0-development",
+  "description": "File name format linter",
+  "main": "index.js",
   "repository": {
-    "url": "git@github.com:usabilla/fnlint.git",
+    "url": "https://github.com/usabilla/fnlint.git",
     "type": "git"
   },
   "author": {
     "name": "Usabilla",
     "email": "development@usabilla.com"
   },
+  "scripts": {
+    "test": "jasmine JASMINE_CONFIG_PATH=jasmine.json",
+    "test:coverage": "./scripts/test.sh",
+    "test:watch": "watch 'npm test' ./lib ./test",
+    "lint": "eslint .",
+    "lint:fn": "./scripts/lint-filenames.js",
+    "lint:fix": "npm run lint -- --fix",
+    "start": "node .",
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+  },
+  "devDependencies": {
+    "@pietvanzoen/jasmine-promise-matchers": "^0.2.1",
+    "@usabilla/js-styleguide": "^1.0.0",
+    "condition-circle": "^1.5.0",
+    "coveralls": "^2.11.15",
+    "eslint": "^3.12.1",
+    "istanbul": "^0.4.5",
+    "jasmine": "^2.5.2",
+    "jasmine-spec-reporter": "^3.1.0",
+    "semantic-release": "^6.3.2",
```

Comment:
Still unsure about the best solution for commit validation. Would like to do more research.